sources/Interop/Windows/um/winnt/EVENTSFORLOGFILE.cs

```diff
@@ -0,0 +1,22 @@
+// Copyright © Tanner Gooding and Contributors. Licensed under the MIT License (MIT). See License.md in the repository root for more information.
+
+// Ported from um/winnt.h in the Windows SDK for Windows 10.0.18362.0
+// Original source is Copyright © Microsoft. All rights reserved.
+
+namespace TerraFX.Interop
+{
+    public unsafe partial struct EVENTSFORLOGFILE
+    {
+        [NativeTypeName("DWORD")]
+        public uint ulSize;
+
+        [NativeTypeName("WCHAR [256]")]
+        public fixed ushort szLogicalLogFile[256];
+
+        [NativeTypeName("DWORD")]
+        public uint ulNumRecords;
+
+        [NativeTypeName("EVENTLOGRECORD []")]
+        public EVENTLOGRECORD* pEventLogRecords;
```

Comment:
Same here, it is an inline fixed sized buffer.

I'm guessing I need to fix something in ClangSharp.

Reply:
See above.